### ISSUE TITLE
Bug: Logging URL objects fails

### ISSUE THREAD

**Issue by whitphx** [fullstack-build/tslog#93]
**Describe the bug**
TSlog version: 3.1.1

Similar to #86.
When logging a `URL` object, an error occurs.

```shell
root@88f15f987cd8:/srv# yarn ts-node src/tslogdebug.ts
yarn run v1.22.5
$ /srv/node_modules/.bin/ts-node src/tslogdebug.ts
2021-03-09 18:30:36.039  INFO [src/tslogdebug.ts:5 Object.<anonymous>] TypeError: Cannot convert undefined or null to object
    at getPrototypeOf (<anonymous>)
    at URL.[nodejs.util.inspect.custom] (internal/url.js:357:9)
    at internal/per_context/primordials.js:23:32
    at formatValue (internal/util/inspect.js:783:19)
    at Object.inspect (internal/util/inspect.js:337:10)
    at Logger._inspectAndHideSensitive (/srv/node_modules/tslog/src/LoggerWithoutCallSite.ts:897:26)
    at /srv/node_modules/tslog/src/LoggerWithoutCallSite.ts:664:18
    at Array.forEach (<anonymous>)
    at Logger.printPrettyLog (/srv/node_modules/tslog/src/LoggerWithoutCallSite.ts:632:30)
    at Logger._handleLog (/srv/node_modules/tslog/src/LoggerWithoutCallSite.ts:348:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**To Reproduce**
Run the code below.
```typescript
import { Logger } from "tslog";

const logger = new Logger();
const url = new URL("http://example.com");
logger.info(url);
```

**Additional context**

**Node.js Version**
v14.16.0 (`node:14.16.0-buster` docker image)

**OS incl. Version**
Debian (buster) docker image on Debian (buster) host.

**Comment by terehov** [comment]
Thank you. I am on it.

**Comment by terehov** [comment]
Fixed in 3.1.2 and should now also be more generic. Thank you once again. 

https://github.com/fullstack-build/tslog/releases/tag/v3.1.2

**Comment by whitphx** [comment]
Thank you for such a quick fix! 😄
